UnitedIncome/components
# HouseIcon Component

### Description
> Please provide a brief description of your changes below.

Shifts the house icon component from the Dashboard into the component library. You can define `x` amount of houses using the `houses` prop, and you can highlight `x` amount of them using the `highlight` prop. You're able to display a single house if that's preferred too. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Take a look at the component in the library!

### Comments
> Any additional comments that will aid with the review process.

You can't customize the fill due to the nature of the component. 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-*)
